sklearnex/covariance/tests/test_incremental_covariance.py

```diff
@@ -41,24 +46,32 @@ def test_sklearnex_partial_fit_on_gold_data(dataframe, queue, dtype):
         result = inccov.partial_fit(X_split_df)
 
     expected_covariance = np.array([[0, 0], [0, 0]])
-    expected_means = np.array([0, 1])
+    expected_means = np.array([0, 0])
+
+    if assume_centered:
+        expected_covariance = np.array([[0, 0], [0, 1]])
+    else:
+        expected_means = np.array([0, 1])
```

Comment:
this is a bit confusing to read - can lines 48-49 be added to the conditionals does that cause error with variable not being initialized?

Reply:
if all expected_means and expected_covariance cant be added inside conditionals then I'd set them for assume_centered False outside the condition and update if assume_centered is True